src/components/list/Directory.astro

```diff
@@ -137,22 +150,26 @@ if (!sortAlphabetically) {
 
 <CardGrid>
   {
-    mainList.map((item) => (
-      <LinkCard
-        title={item.entry.data.title}
-        href={`/${item.slug}`}
-        description={item.entry.data.description}
-      />
-    ))
+    mainList.map((item) => {
+      return (
+        <CustomLinkCard
+          title={item.entry.data.title}
+          href={`/${item.slug}`}
```

Comment:
For some reason some cards (e.g. `stronghold`, `updater`) when you click on them the link is broken, not sure if it is related to the lack of a trailing slash.

Reply:
> not sure if it is related to the lack of a trailing slash

probably it is, will add just as a failsafe. It works here btw

Reply:
@dreyfus92 I'm not gonna try the icons you suggested, there's some other improvements that could be done but I think it's better to ship it. Could you please merge if you don't have any other suggestion